rules/create-draft-pull-requests/rule.md

```diff
@@ -0,0 +1,48 @@
+---
+type: rule
+archivedreason: 
+title: Pull Requests - Do you use Draft Pull Requests?
```

Comment:
Hi @SpookyBoogy2016 

I think this rule is more about indicating draft PRs rather than using them

- [ ] 1. What about changing the title to "Do you use and indicate Draft Pull Requests?"

Reply:
Makes sense. Resolved.